server_update.py

```diff
@@ -1029,6 +1029,7 @@ def get_new(self, default_version='latest', default_build='latest'):
     parser.add_argument('-i', '--interactive', help='Prompts the user for the version they would like to install.', action='store_true')
     parser.add_argument('-nlc', '--no-load-config', help='Will not load config information', action='store_false')
     parser.add_argument('-ndc', '--no-dump-config', help="Will not dump configuration information.", action='store_false')
+    parser.add_argument('-p', '--post-update-command', help="Will execute this command once the update is completed.", default="NONE")

```

Comment:
This argument only takes one parameter, when it should take multiple. This can be done using the nargs parameter.